frontend/webadmin/modules/webadmin/src/main/java/org/ovirt/engine/ui/webadmin/widget/host/HostNetworkInterfaceListViewItem.java

```diff
@@ -182,8 +182,9 @@ private IsWidget createRxColumn(VdsNetworkInterface hostInterface) {
         divider.addStyleName(RATE_DIVIDER);
         rxCol.add(divider);
         dl = Document.get().createDLElement();
-        addDetailItem(templates.sub(constants.rxTotal(), constants.bytes()),
-                SafeHtmlUtils.fromString(totalRenderer.render(hostInterface.getStatistics().getReceivedBytes())), dl);
+        String rendered = hostInterface.getStatistics().getReceivedBytes() != null ?
+            totalRenderer.render(hostInterface.getStatistics().getReceivedBytes()) : constants.unAvailablePropertyLabel();
```

Comment:
Is this needed? As Lucia mentioned the `RxTxTotalRenderer` is capable of handling `null`.

Reply:
Well, apparently it does not, because it was used before the change and still an NPE was thrown. I will debug and try to find the failure point.

Reply:
Using the debugger on the pre-change code sheds produces a stacktrace with more java bindings. This revealed that the NPE is thrown from a different line in the code than I initially suspected. Changed the commit accordingly. Also, indeed the renderer assigns 'N\A' without any modification as Lucia pointed out.

Reply:
for documentation here is the exception from the ui.log:
```
2022-03-28 18:16:22,066+03 ERROR [org.ovirt.engine.ui.frontend.server.gwt.OvirtRemoteLoggingService] (default task-7) [] Uncaught exception: com.google.gwt.event.shared.UmbrellaException: Exception caught: (Type
Error) : com_google_gwt_lang_Cast_castTo__Ljava_lang_Object_2Lcom_google_gwt_core_client_JavaScriptObject_2Ljava_lang_Object_2(...).getReceiveDrops__Ljava_math_BigInteger_2() is null
        at Unknown.java_lang_Throwable_Throwable__Ljava_lang_String_2Ljava_lang_Throwable_2V(webadmin-0.js)
        at Unknown.java_lang_Exception_Exception__Ljava_lang_String_2Ljava_lang_Throwable_2V(webadmin-0.js)
        at Unknown.java_lang_RuntimeException_RuntimeException__Ljava_lang_String_2Ljava_lang_Throwable_2V(webadmin-0.js)
        at Unknown.com_google_web_bindery_event_shared_UmbrellaException_UmbrellaException__Ljava_util_Set_2V(webadmin-0.js)
        at Unknown.com_google_gwt_event_shared_UmbrellaException_UmbrellaException__Ljava_util_Set_2V(webadmin-0.js)
        at Unknown.com_google_gwt_event_shared_HandlerManager_fireEvent__Lcom_google_gwt_event_shared_GwtEvent_2V(webadmin-0.js)
        at Unknown.com_google_gwt_view_client_SelectionModel$AbstractSelectionModel_fireEvent__Lcom_google_gwt_event_shared_GwtEvent_2V(webadmin-0.js)
```